docs/docs/apps/README.md

```diff
@@ -8,6 +8,34 @@ But Pipedream-integrated apps provide a few benefits:
 - Pipedream provides [pre-built actions](/components#actions) that wrap common operations for the app. You shouldn't have to write the code to send a message to Slack, or add a new row to a Google Sheet, so actions make that easy. Actions are just code, so you can fork and modify them, or even [publish your own to the Pipedream community](/apps/contributing/).
 - [You have access to your API keys and access tokens in code steps](/code/nodejs/auth/), so you can write any code to authorize custom requests to these apps.
 
+## Premium Apps
+
+The vast majority of integrated apps on Pipedream are free to use in your workflows across any plan. However, in order to use any of the below apps in an active workflow, you'll need to be on the Advanced plan or higher:
```

Comment:
```suggestion
The vast majority of integrated apps on Pipedream are free to use in your workflows across any plan. However, to use any of the apps below in an active workflow, you need to be on the Advanced plan or higher:
```

Reply:
Removed "in order" and "you'll" to follow Google's Developer docs style guide (conciseness and contractions)